code/modules/clothing/gloves/special.dm

```diff
@@ -49,6 +50,16 @@
 
 	QDEL_NULL(pull_component_weakref)
 
+/obj/item/clothing/gloves/cargo_gauntlet/security
+	name = "\improper B.R.I. gauntlets"
+	desc = "These heavy duty gauntlets can latch on to handcuffs making them very challenging to escape. Their bulky design might make firing guns a bit harder though."
```

Comment:
Not that I care but security has tackling gloves as an alternative to this, not sure what the point of the nerf on shooting is meant to accomplish. If anything it would encourage security to even further only use melee weapons.

Reply:
The way I see it, tackling gloves are an upgrade while these are a sidegrade, I'd be open for a different downside if you've got an idea though.